onsite/phosphors/cli.py

```
@@ -237,9 +240,26 @@ def phosphors(
+                                "phospho_decoy_count",
+                                "PhosphoRS_pep_score",
+                                "PhosphoRS_site_probs",
+                                "PhosphoRS_site_delta",
+                                "SpecEValue_score",
+                                "ProForma",
+                            ]:
+                                if failed_hit.metaValueExists(k):
+                                    try:
+                                        failed_hit.removeMetaValue(k)
+                                    except Exception:
+                                        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)